gateways/orbital/request_builders.go

```diff
@@ -51,10 +55,13 @@ func buildCaptureRequest(captureRequest *sleet.CaptureRequest) Request {
 	}
 
 	body.XMLName = xml.Name{Local: RequestTypeCapture}
+	body.OrbitalConnectionUsername = credentials.Username
```

Comment:
wait why dont we just move this up into the initialization of the struct?

Reply:
moved